iree/samples/simple_embedding/BUILD

```diff
@@ -115,17 +139,15 @@ endif()
 
 run_binary_test(
     name = "simple_embedding_dylib_test",
-    args = ["dylib"],
-    test_binary = ":simple_embedding_run",
+    test_binary = ":simple_embedding_dylib",
```

Comment:
Nit: while you're here, could you add the appropriate driver tags to each of these tests?

"driver=vulkan", "driver=dylib", etc.

Reply:
```suggestion
    test_binary = ":simple_embedding_dylib",
    tags = ["driver=dylib"],
```

Reply:
run_binary_test doesn't support the tags field yet. :( https://github.com/google/iree/blob/7c2d310/build_tools/bazel_to_cmake/bazel_to_cmake_converter.py#L703

Can add this in a following PR.